### PR TITLE
[Android] Fix Switch Thumb wrong color

### LINKED ISSUE #7249: [Bug] Android Switch Slider wrong color?
### Description

After updating from 4.1.0 (first service release) to 4.2.0, the switch on Android has a strange behavior. If I switch them, the circle stays white which was not the case before and is not the case when I reload the page, see: https://i.imgur.com/sWVbdWX.png - All Orange Switches where on IsToggles while the page has loaded, the white ones just got toggled after the page loads. 

This bug is only happening for the Switch itself. It does not occur with SwitchCell controls. 

### Steps to Reproduce

1. Update to 4.2.0

### Expected Behavior

The Switch should not have different color. If this was an intended behvior change it should be equal accross all switches. 

### Actual Behavior

Different if the switch was loaded with a true value for IsToggled or if it was set after the page loads.

### Basic Information

- Version with issue:
- Last known good version: 4.1.0 (first service release)
- IDE:
- Platform Target Frameworks: 
  - Android: Latest stable
- Android Support Library Version: 
- Nuget Packages:
- Affected Devices:

### PR DESCRIPTION
**Pull request by jsuarezruiz**
### Description of Change ###

Fix **Switch Thumb wrong color on Android**. We introduce an option to change the ThumbColor here https://github.com/xamarin/Xamarin.Forms/pull/6798
We save in a variable the color of the Thumb https://github.com/xamarin/Xamarin.Forms/blob/master/Xamarin.Forms.Platform.Android/Renderers/SwitchRenderer.cs#L87 and use it if ThumbColor is not set https://github.com/xamarin/Xamarin.Forms/blob/master/Xamarin.Forms.Platform.Android/Renderers/SwitchRenderer.cs#L141
The problem is that the default Thumb color is different if the switch is Checked or not (white or pink respectively). Applied changes to set the color of the ThumbColor only when it is not Color.Default. 

### Issues Resolved ### 

- fixes #7249

### API Changes ###
 
 None

### Platforms Affected ### 

- Android

### Behavioral/Visual Changes ###

None

#### Before
7249-before

#### After
7249-after

### PR Checklist ###

- [ ] Targets the correct branch
- [x] Tests are passing (or failures are unrelated)

### CODE REVIEW

**Comment by samhouts** [comment]
@jsuarezruiz What happens if you have a custom switch renderer that sets the color? Can you add a test case for that to make sure we won't break those? Thanks!

File: Xamarin.Forms.ControlGallery.Android/Issue7249SwitchRenderer.cs [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+using Android.OS;
+using Xamarin.Forms.Platform.Android;
+using Xamarin.Forms.Controls.Issues;
+using static Android.Widget.CompoundButton;
+
+namespace Xamarin.Forms.ControlGallery.Android
+{
+	public class Issue7249SwitchRenderer : SwitchRenderer
+	{
+		private Issue7249Switch _view;
```
Comment (samhouts) [inline Xamarin.Forms.ControlGallery.Android/Issue7249SwitchRenderer.cs]:
```suggestion
		Issue7249Switch _view;
```

**Comment by samhouts** [comment]
@jsuarezruiz Can you rebase and fix the build? Thanks!

**Comment by samhouts** [comment]
@jsuarezruiz Sorry to ask again, but can you retarget to 4.4.0? Thanks!!

**Comment by jsuarezruiz** [comment]
Retarget to 4.4.0.

**Pull request merged**